### DISCUSSION TITLE
Adding Elastic Fleet to SO Kibana

### DISCUSSION THREAD

**Discussion by CallumBasham** [Security-Onion-Solutions/securityonion#6468] [Unsupported Versions] [answered]
I noticed when looking at my instance of Kibana on Security onion, it is missing common features seen in tutorials/videos and other screenshots: https://imgur.com/a/EcYi1kA. 

What I'm looking for is Management > Fleet, which is the ELK's feature for managing endpoint agent logs: 
https://www.elastic.co/guide/en/fleet/current/fleet-overview.html
https://www.elastic.co/guide/en/fleet/current/fleet-server.html

Any ideas on how to add this feature into my SO's ELK? 

Thanks!

**Comment by njohnson-q** [comment, marked as answer]
The features are still there but hidden by default. To add go to 'Stack Management' in Kibana, click 'Spaces' on the left hand side. Enter the default space and click 'Management' and enable the checkmark next to fleet and save.

**Reply by CallumBasham** [reply]
That's brilliant thank you so much!
